closed/OpenJ9.gmk

```diff
@@ -28,6 +28,14 @@ ifeq (,$(BUILD_ID))
   BUILD_ID := 000000
 endif
 
+# 64 bit compressedrefs builds put some of the libraries in lib/compressedrefs.
+# What's the appropriate directory for a 32 bit build? I've opted for 'default'.
+ifeq (1,$(OPENJDK_DOCKER_CC_ARM))  # condition should be more specific to 32bit
```

Comment:
This limits to arm only, could this not be set at the time platform is determined within the custom-hook-m4?

Reply:
Hi Joe, my objective with this set of changes is to take the shortest path that isn't totally objectionable to getting a working arm build. For sure there will be better solutions to most of the problems, and where that's obvious I'm adding comments to the effect. I'd like to get these changes in so that we can have the platform building and testing (and therefore preventing ever more bit-rot in the arm code base). I would suggest we open issues for future improvements and then people that actually understand how the build system works can implement gold standard solutions.